src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -87,6 +87,31 @@ public RequestSpecification generateBearerTokenForNonPuiManager() {
         }
     }
 
+    public RequestSpecification generateBearerTokenForPuiCaa() {
+        Map<String, Object> response = professionalApiClient.createOrganisation();
+        String orgIdentifierResponse = (String) response.get("organisationIdentifier");
+        professionalApiClient.updateOrganisation(orgIdentifierResponse, hmctsAdmin);
+
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add("pui-caa");
+        String userEmail = randomAlphabetic(5).toLowerCase() + "@hotmail.com";
+        String lastName = "someLastName";
+        String firstName = "someName";
+
+        bearerTokenForPuiUserManager = professionalApiClient.getMultipleAuthHeadersExternal(puiCaa, firstName, lastName, userEmail);

```

Comment:
Could you change this variable name to `bearerTokenForPuiCaa` instead of `bearerTokenForPuiUserManager` just to keep in line with the method name

Reply:
sure will do